daml/Marketplace/Custody/Role.daml

```diff
@@ -79,11 +83,14 @@ template Request
     signatory provider
 
     controller operator can
-      Approve : ContractId Role
+      Approve : (ContractId Role, ContractId Issuance.Service, ContractId Custody.Service)
         with
           observers : Set Party
         do
-          create Role with ..
+          custodyCid    <- createOrLookup Custody.Service with customer = provider, ..
+          issuanceCid   <- createOrLookup Issuance.Service with customer = provider, ..
+          roleCid <- createOrLookup Role with ..
```

Comment:
the alignment could be fixed here...it's not 100% necessary to align on `<-` at all times - especially when one variable/pattern match in the group is particularly long - but since the ones above are aligned I think it would look nicer for `roleCid` to be too